### PR TITLE
Mempool updated to use js procedures

### PR DESCRIPTION
**Pull request by cryptodread**
Updated the config, properties and dependencies to use the js procedures in 0.3.1.

### CODE REVIEW

File: scripts/embassy.js [hunk 1]
```diff
@@ -0,0 +1,4298 @@
+// deno-fmt-ignore-file
```
Comment (Blu-J) [inline scripts/embassy.js:1]: Warn: I usually don't commit the embassy.js

File: scripts/embassy.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// @ts-check
```
Comment (Blu-J) [inline scripts/embassy.ts:1]:
```suggestion
```
Don't need this line, it was for js files orginally and I messed up  the first ones

**Review by Blu-J — APPROVED** [review]
Ts looks good, Not sure if logic is matching the prrevious specs, but we should test those.

**Review by Blu-J — APPROVED** [review]

File: scripts/services/setConfig.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  });
+  await effects.writeFile({
+    path: "start9/config.yaml",
+    toWrite: YAML.stringify(newConfig),
+    volumeId: "main",
+  });
+
+  const result: SetResult = {
+    signal: "SIGTERM",
+    "depends-on": {},
```
Comment (Blu-J) [inline scripts/services/setConfig.ts:27]: Need to work in the file from 
https://github.com/Start9Labs/mempool-wrapper/blob/feat/js/assets/compat/dependencies.yaml

Comment (Blu-J) [inline scripts/services/setConfig.ts:27]:
```suggestion
    "depends-on": {
       ...dependsOnElectrs,
       ...dpenedsOnBitcoind,
    },
```

File: scripts/embassy.js (hunk 1 above)
Comment (cryptodread) [reply scripts/embassy.js:1]: added this to .gitignore

**Review by Blu-J — APPROVED** [review]

**Review by Blu-J — APPROVED** [review]

**Review by elvece — APPROVED** [review]

**Pull request merged**